sdk/core/core/src/az_span_builder.c

```diff
@@ -23,6 +25,59 @@ AZ_NODISCARD az_result az_span_builder_append_byte(az_span_builder * const self,
   return AZ_OK;
 }
 
+AZ_INLINE uint8_t _az_decimal_to_ascii(uint8_t const d) {
+  assert(d < 10);
+  return '0' + d;
+}
+
+AZ_INLINE AZ_NODISCARD az_result
+_az_span_builder_append_unsigned_number(az_span_builder * const self, unsigned long long const n) {
```

Comment:
```suggestion
_az_span_builder_append_unsigned_number(az_span_builder * const self, uint64_t const n) {
```

Reply:
The intention here is to handle `unsigned long long`, the biggest integer type you can get using the standard syntax, regardless of the actual "bitness" - the code will handle any (well, 3 bits minimum)

Reply:
Done